src/Sylius/Bundle/CoreBundle/Test/Factory/TestUserFactory.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Test\Factory;
+
+use Sylius\Component\Resource\Factory\FactoryInterface;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class TestUserFactory implements TestUserFactoryInterface
+{
+    const DEFAULT_USER_EMAIL = 'john.doe@example.com';
```

Comment:
But you are not using them at the moment. According to YAGNI, I would vote to keep them private.   If one has to use them, he will change them to constants. WDYT?